SECP-service/src/main/java/com/visucius/secp/Controllers/User/UserRegistrationController.java

```diff
@@ -42,7 +42,7 @@ public UserRegistrationResponse registerUser(UserRegistrationRequest request) {
         {
             try {
                 String hashPassword = PasswordUtil.createHash(request.password);
-                User user = new User(request.firstName, request.lastName, request.userName, request.email, hashPassword);
+                User user = new User(request.firstName, request.lastName, request.userName, request.displayName, request.email, hashPassword, request.avatar_url);
```

Comment:
the constructor is getting too long. Don't add new parameters to the constructor. just use the set method.